.travis.yml

```diff
@@ -3,20 +3,21 @@ language: python
 python:
   - "2.7"
 
+env:
+  - LINT_CHECK="1"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1" LINT_CHECK="0"
```

Comment:
@pedrobaeza I've used the template from https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml. What would be better?